step/step.go

```diff
@@ -49,7 +49,7 @@ func (s EASBuilder) ProcessConfig() (Config, error) {
 	var input Input
 	err := s.inputParser.Parse(&input)
 	if err != nil {
-		return Config{}, err
+		return Config{}, fmt.Errorf("Process config: %w", err)
```

Comment:
Why not in the main package's run func: https://github.com/bitrise-steplib/bitrise-step-run-eas-build/blob/main/main.go#L17

so that you do not need to return an upper cased error + no need for adding the top-level context in lower levels.